build.gradle

```diff
@@ -2,8 +2,11 @@
 plugins {
     id 'java-library'
     id "org.openmicroscopy.project" version "5.5.0-m5"
+    id "application"
```

Comment:
We don't need this plugin for this project.

Reply:
This is required, due to ``mainClassName = "omero.gateway.Gateway"``